### ISSUE TITLE
Compiling fails for typescript-json-schema.ts file

### ISSUE THREAD

**Issue by ALJCepeda** [YousefED/typescript-json-schema#484]
Importing fails when `strictNullChecks === true`

![image](https://user-images.githubusercontent.com/5140822/170917367-90a6b7af-a977-462b-a5df-164757b6b02f.png)

![image](https://user-images.githubusercontent.com/5140822/170917393-016d1438-95f0-41b1-af9b-19be26aa9df6.png)

**Comment by domoritz** [comment]
You can set strict null checks to off for now.

**Comment by ALJCepeda** [comment]
> You can set strict null checks to off for now.

Then I can't have strict null check for the rest of my project. This seems like an easy fix for ya'll

**Comment by domoritz** [comment]
Yeah, it would be nice to support strict null checks. I'd be happy to review a pull request.
